db/migrate/20180403144853_convert_vm_attrs_to_hash.rb

```diff
@@ -0,0 +1,80 @@
+class ConvertVmAttrsToHash < ActiveRecord::Migration[5.1]
+  def up
+    transform :to_h
+  end
+
+  def down
+    transform :to_params
+  end
+
+  YML_HASH = '!ruby/hash:ActiveSupport::HashWithIndifferentAccess'
+  YML_PARAMS_HASH = '!ruby/hash:ActionController::Parameters'
+  YML_PARAMS_OBJ = '!ruby/object:ActionController::Parameters'
+  YML_PARAMS_IVARS = '!ruby/hash-with-ivars:ActionController::Parameters'
```

Comment:
Maybe we should combine all of these into `/\!ruby\/[\w-]+:ActionController::Parameters/`? gsub knows how to check for regex as well, not sure about the preformance impact.